licensing/third-party-programs.txt

```diff
@@ -345,3 +345,53 @@ DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
 THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
 (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+-------------------------------------------------------------
+
+6. fatchord/WaveRNN
+   MIT License
+
+Copyright (c) 2019 fatchord (https://github.com/fatchord)
```

Comment:
These are in the wrong order: the copyright notice goes under the project name, and the license name afterwards.

In addition, if this is just the standard MIT license, you should merge this entry with entry 2 (you can list multiple projects per entry if they have the same license).